frontend/javascripts/features.js

```diff
@@ -19,6 +19,7 @@ export function getDemoDatasetUrl() {
     throw new Error("Features not yet loaded.");
   }
 
+  // todop: how to pass the fallback layer here?
```

Comment:
todo. we should probably encode the entire url in `application.conf` (so that it includes `createExplorative/hybrid?fallbackLayerName=...`).